archunit/src/main/java/com/tngtech/archunit/core/domain/properties/HasParameterTypes.java

```diff
@@ -30,6 +31,9 @@
 
 public interface HasParameterTypes {
 
+    @PublicAPI(usage = ACCESS)
+    List<JavaType> getParameterTypes();
```

Comment:
Should we add some JavaDoc to disambiguate `getParameterTypes()` from `getTypeParameters()` (which also exists in `JavaCodeUnit`)?

Reply:
Good idea, I added some Javadoc with examples! :+1: